server/hotrod/src/main/java/org/infinispan/server/hotrod/Decoder2x.java

```diff
@@ -411,6 +411,13 @@ public void customReadKey(HotRodHeader header, ByteBuf buffer, CacheDecodeContex
             }).isPresent()) {
                return;
             }
+            if (Constants.isVersionPost25(header.version)) {
+               int listenerInterests = ExtendedByteBufJava.readMaybeVInt(buffer);
+               if (listenerInterests == Integer.MIN_VALUE)
+                  return;
+
+               requestCtx.setListenerInterests(listenerInterests);
+            }
```

Comment:
Seems good to me.  Yeah like I mentioned it seems like the tests aren't using it properly atm.  Either that or we would have gotten failures on another call.